cpp/log/cert.h

```diff
@@ -148,12 +150,21 @@ class Cert {
   // Returns ERROR if the cert is not loaded.
   util::Status PublicKeySha256Digest(std::string* result) const;
 
+  // Sets the Subject Alternative Name dNSNames in |dns_alt_names|.
+  // Returns util::Status::OK if the SAN dNSNames were extracted.
+  // Returns ?? if the DAN dNSNames could not be extracted.
+  // Returns ?? if the cert is not loaded.
+  util::Status SubjectAltNames(std::vector<std::string>* dns_alt_names) const;
```

Comment:
I was in two minds about util::Status Fn(stuff, Thing* out)  and util::StatusOr<Thing> Fn(stuff)
Actually I might make the SPKI method be the former to be consistent with everything else first.